modules/admin_manual/pages/configuration/server/config_sample_php_parameters.adoc

```diff
@@ -829,6 +829,52 @@ that triggers auto purging of deleted files for this user
 'trashbin_purge_limit' => 50,
 ....
 
+== Define trashbin directory skip list
+Define a list of directories that will skip the trashbin and therefore be deleted immediately.
+
+Only defined directories and only in the root of a mount will skip the trashbin.
+Consider not to use reserved directory names when using snapshot capable storage systems.
+The setting expects folder names with or without trailing slash.
+All the content of such directories including their subdirectories will also skip the trashbin.
+
+
+'trashbin_skip_directories' => [
+'temp',
+],
+
+=== Define trashbin file extension skip list
+Define a list of file extensions to determine files that will skip the trashbin and therefore be deleted immediately.
+
+Extension names are valid for all mount points, take care when selecting the names.
+
+Values must not have a leading ".", otherwise corresponding files wont be detected.
+
+==== Code Sample
+
+[source,php]
+....
+'trashbin_skip_extensions' => [
+	'iso',
+	'mkv',
+],
+....
+
+=== Define trashbin threshold size
+Define a threshold for files to skip the trashbin and delete immediately
+Once the size of a resource is greater than or equal the given value, the trashbin will be skipped.
+
+File sizes are valid for all mount points, take care when defining the threshold.
```

Comment:
This is confusing. It tries to warn me of something, but I don't see what exactly the problematic situation is.
I'd be more surprised if file sizes would not apply to all storages -- do we have an example what exactly a user should look for and take care?
Is a resource different than a file?

Reply:
In general, this is a transformation of content coming from core. Any changes wanted must be made in core and post transformed to docs (a `config-to-docs` run).
Means, this can safely be merged and any updates from core can be post-transferred.

Reply:
Could be improved like:
`File sizes are valid for all mount points...` -->
`File sizes are valid for all files in all mount points...`